katello.gemspec

```diff
@@ -56,7 +56,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency "pulp_container_client", ">= 2.0.0", "< 2.2.0"
   gem.add_dependency "pulp_deb_client", ">= 2.6.0", "< 2.8.0"
   gem.add_dependency "pulp_rpm_client", ">=3.10.0", "< 3.11.0"
-  gem.add_dependency "pulp_2to3_migration_client", ">= 0.8.0", "< 1.0.0"
+  gem.add_dependency "pulp_2to3_migration_client", ">= 0.8.0", "< 0.12.0"
```

Comment:
oh hah, i guess bumping isn't even needed.  I forgot we bumped this max to 1.0  for easier upgrades.  Lets just close this!